tests/serverpod_test_server/test_integration/database_operations/entity_relations/entities_with_list_relations/order_by_test.dart

```diff
@@ -47,12 +116,45 @@ void main() async {
 
   group('Given ordered find query', () {
     test(
-        'when ordering on filtered count of nested many relation then result is as expected.',
+        'when ordering on filtered count of nested many relation then result order is as expected.',
+        () async {
+      var persons = await Person.db.find(
+        session,
+        // Order people by number of people in their organization ascending
+        // and then their name.
```

Comment:
Suggestion: Put this comment in the test description instead!

Reply:
Updated these tests with new descriptions. I don't think the implementation detail is important for the test scenario. Check out the new descriptions and let me know what you think.